permissions/plugin-pull-request-monitoring.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "pull-request-monitoring"
+github: "simonsymhoven/pull-request-monitoring-plugin"
```

Comment:
We don't publish plugins outside of the jenkinsci github org, this needs to go through the hosting process: https://www.jenkins.io/doc/developer/publishing/requesting-hosting/